package.json

```diff
@@ -30,8 +33,11 @@
     "@nuxt/core": "^2.15.8",
     "@nuxtjs/axios": "^5.13.6",
     "@nuxtjs/dotenv": "^1.4.0",
+    "cypress": "^8.3.1",
```

Comment:
These dependencies should not be listed in `dependencies`. They should at least be in `devDependencies` or even better in `optionalDependencies`.